web/e2e-tests/message-basics.test.ts

```diff
@@ -104,15 +125,16 @@ async function expect_recent_view(page: Page): Promise<void> {
 async function test_navigations_from_home(page: Page): Promise<void> {
     return; // No idea why this is broken.
     console.log("Narrowing by clicking stream");
-    await page.click(`#zhome [title='Narrow to stream "Verona"']`);
+    // TODO: Ensure we are checking the correct message list.
+    await page.click(`.message-list [title='Narrow to stream "Verona"']`);
```

Comment:
For this detail, we could consider using a `:visible` selector. We could also consider having a `.current-message-list` class that gets toggled on/off when the list is marked as current.

Reply:
Oh we already have `.focused-message-list` for this purpose.

Reply:
Thanks, removed the TODO.